.travis.yml

```diff
@@ -109,14 +109,19 @@ matrix:
       env: allowTestFail=true
       os: linux
       addons:
+        artifacts:
+          s3_region: "us-west-2"
```

Comment:
Could you please share some explanation of what these do?

Reply:
I'm adding some debugging stuff. Trying to figure out why build fails in travis. Please ignore changes in this branch right now.